src/rt/trace.d

```diff
@@ -70,23 +68,24 @@ struct Stack
     timer_t subtime;            // time used by all subfunctions
 }
 
-__gshared                       // doesn't work with multithreaded code anyway
+static
 {
     Symbol* root;               // root of symbol table
 
     Stack* stack_freelist;
     Stack* trace_tos;           // top of stack
-    int trace_inited;           // !=0 if initialized
-    timer_t trace_ohd;
+    bool trace_inited;
+}
 
-    Symbol** psymbols;
-    uint nsymbols;              // number of symbols
+__gshared
+{
+    Symbol* groot;              // merged symbol table
+    int gtrace_inited;          // !=0 if initialized
```

Comment:
`bool`? or are you worried about word tearing? Also: would be nice if the order were the same in the TLS and global.